lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -242,12 +284,16 @@ sub call_rpc {
 
 =head2 request
 
-Sends request to backend service. The method accepts single unnamed argument:
+Sends request to backend service. The method accepts following arguments:
 
 =over 4
 
 =item * C<request_data> - an C<arrayref> containing data for the item which is going to be put into redis stream.
 
+=item * C<category_name> - this will be passed to C<_send_request> to specify which redis category this message belongs to.
```

Comment:
A named parameter would be `category_name`, `request_data` - *unnamed* parameters, i.e. positional variables, need the `$` prefix for clarity.

```suggestion
=item * C<$category_name> - this will be passed to C<_send_request> to specify which redis category this message belongs to.
```